src/qt/transactionview.h

```diff
@@ -69,6 +70,9 @@ class TransactionView : public QWidget
     QLineEdit *addressWidget;
     QLineEdit *amountWidget;
 
+    QTimer *m_inputDelayPrefix;
```

Comment:
Hmm.. I see. My advice is against the developer notes then. It just looked confusing to have two single instance variables with sneak case where the rest is without `m_` and camel case. But I guess we have to start somewhere to adopt the style rules.